_i18n/de.yml

```diff
@@ -488,7 +488,7 @@ press-kit:
   whyconf1: Wir empfehlen jedem, Community-Mitglieder aus dem größten Datenschutzprojekt
     in verwandte Nachrichtenartikel und Veranstaltungen einzubeziehen.
   altpk-2: Bild von fluffypony während einer Präsentation vor Publikum auf der DEFCON
-  altpk-1: Bild einer Frau, die vor einem "Monero Talk"-Banner steht
+  altpk-1: Bild einer Frau, die vor einem Monero Talk-Banner steht
```

Comment:
Just a small nitpick, but in German it would look better with two hyphens like so: "Monero-Talk-Banner"

Reply:
This format was chosen by german translators, we can always suggest a new translation on weblate (after it will be up again) :)

Reply:
Ok. Has nothing to to with the problem the PR wants to solve anyway!